radixdlt-cli/build.gradle

```diff
@@ -30,14 +30,16 @@ buildscript {
 apply plugin: 'java'
 apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'groovy'
+apply plugin: 'application'
 repositories {
-    jcenter()
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
-    maven { url 'https://jitpack.io' }
+	jcenter()
```

Comment:
Should use spaces instead of tabs here for consistency.

Reply:
Actually, it doesn't look like any of this actually changed, so should be reverted.